src/testdir/dumps/Test_pum_with_preview_win.dump

```diff
@@ -1,12 +1,12 @@
-|1+0&#ffffff0|i|n|f|o| @69
+|2+0&#ffffff0|i|n|f|o| @69
 |~+0#4040ff13&| @73
 |~| @73
 |[+1#0000000&|S|c|r|a|t|c|h|]| |[|P|r|e|v|i|e|w|]| @37|1|,|1| @11|A|l@1
-|o+0&&|n|e> @71
-|o+0#0000001#e0e0e08|n|e| @11| +0#4040ff13#ffffff0@59
-|t+0#0000001#ffd7ff255|w|o| @11| +0#4040ff13#ffffff0@59
+|t+0&&|w|o> @71
```

Comment:
emm seems like not correct.. wired i tried with minimal test.vim and build vim i can't reproduce it ?